.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Crate Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+
+### Feature Categories:
+
+
+- [ ] Express
+- [ ] React
+- [ ] Styling/CSS
+- [ ] Documentation
+- [ ] Other: 
+
+**Describe the feature/Bug your gonna work on**
```

Comment:
** you're 😄 
Beyond that, looks great. We should discuss what kinds of labels we want to use!

Reply:
In addition, (although not directly related) we should norm on Branch naming conventions too!

Reply:
You mean you don't want me to name all my branches some jumble of 
'sienna-fix-the-thing' hahaha. For real though, down on this!

Reply:
Love the format!